highspy/highs_bindings.cpp

```diff
@@ -929,4 +929,198 @@ PYBIND11_MODULE(highspy, m) {
+      .value("kSimplexInfeasibilityProofRefinementNo",
+             SimplexPivotalRowRefinementStrategy::
+                 kSimplexInfeasibilityProofRefinementNo)
+      .value("kSimplexInfeasibilityProofRefinementUnscaledLp",
+             SimplexPivotalRowRefinementStrategy::
+                 kSimplexInfeasibilityProofRefinementUnscaledLp)
+      .value("kSimplexInfeasibilityProofRefinementAlsoScaledLp",
+             SimplexPivotalRowRefinementStrategy::
+                 kSimplexInfeasibilityProofRefinementAlsoScaledLp)
+      .value("kSimplexInfeasibilityProofRefinementMax",
+             SimplexPivotalRowRefinementStrategy::
+                 kSimplexInfeasibilityProofRefinementMax)
+      .export_values();
+  py::enum_<SimplexPrimalCorrectionStrategy>(simplex_constants,
+                                             "SimplexPrimalCorrectionStrategy")
+      .value(
+          "kSimplexPrimalCorrectionStrategyNone",
+          SimplexPrimalCorrectionStrategy::kSimplexPrimalCorrectionStrategyNone)
+      .value("kSimplexPrimalCorrectionStrategyInRebuild",
+             SimplexPrimalCorrectionStrategy::
+                 kSimplexPrimalCorrectionStrategyInRebuild)
+      .value("kSimplexPrimalCorrectionStrategyAlways",
+             SimplexPrimalCorrectionStrategy::
+                 kSimplexPrimalCorrectionStrategyAlways)
+      .export_values();
+  py::enum_<RebuildReason>(simplex_constants, "RebuildReason")
+      .value("kRebuildReasonCleanup", RebuildReason::kRebuildReasonCleanup)
+      .value("kRebuildReasonNo", RebuildReason::kRebuildReasonNo)
+      .value("kRebuildReasonUpdateLimitReached",
+             RebuildReason::kRebuildReasonUpdateLimitReached)
+      .value("kRebuildReasonSyntheticClockSaysInvert",
+             RebuildReason::kRebuildReasonSyntheticClockSaysInvert)
+      .value("kRebuildReasonPossiblyOptimal",
+             RebuildReason::kRebuildReasonPossiblyOptimal)
+      .value("kRebuildReasonPossiblyPhase1Feasible",
+             RebuildReason::kRebuildReasonPossiblyPhase1Feasible)
+      .value("kRebuildReasonPossiblyPrimalUnbounded",
+             RebuildReason::kRebuildReasonPossiblyPrimalUnbounded)
+      .value("kRebuildReasonPossiblyDualUnbounded",
+             RebuildReason::kRebuildReasonPossiblyDualUnbounded)
+      .value("kRebuildReasonPossiblySingularBasis",
+             RebuildReason::kRebuildReasonPossiblySingularBasis)
+      .value("kRebuildReasonPrimalInfeasibleInPrimalSimplex",
+             RebuildReason::kRebuildReasonPrimalInfeasibleInPrimalSimplex)
+      .value("kRebuildReasonChooseColumnFail",
+             RebuildReason::kRebuildReasonChooseColumnFail)
+      .value("kRebuildReasonForceRefactor",
+             RebuildReason::kRebuildReasonForceRefactor)
+      .value("kRebuildReasonExcessivePrimalValue",
+             RebuildReason::kRebuildReasonExcessivePrimalValue)
+      .value("kRebuildReasonCount", RebuildReason::kRebuildReasonCount)
+      .export_values();
```

Comment:
I suppose these could be dropped from the Python API.

Reply:
Yes, these are utterly irrelevant to users